dev-docs/bidders/between.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: bidder
-title: BetweenDigital
-description: between bid adapter
+title: Between
+description: Prebid Between Bidder Adaptor
 pbjs: true
 biddercode: between
+media_types: banner
 ---
 
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
-| Name       | Scope    | Description                               | Example   |
-|------------+----------+-------------------------------------------+-----------|
-| `w`        | required | width of placement(Number)                | 240       |
-| `h`        | required | height of placement(Number)               | 400       |
-| `s`        | required | Section ID, provided by betweendigital    | 12345     |
+| Name          | Scope    | Description | Example | Type     |
+|---------------|----------|-------------|---------|----------|
+| `host` | required |  between host url         | `ru.betweendigital.com/s2s`   | `string` |
```

Comment:
The w, h, and s values are from the PBJS adapter. Do those no longer apply? .. or perhaps the PBS and PBJS adapters are implemented very differently?

Reply:
yes, PBS and PBJS are almost not related as of now

Reply:
Gotcha. This doc is intended to cover both adapters. Please consider 2 bid params tables, one for PBJS and one for PBS. Please update the host example to match the updated usage of subdomian.